doc/de.po

```diff
@@ -1116,12 +1117,20 @@ msgstr "PROXY UNTERSTÜTZUNG"
 # type: Plain text
 #. type: Plain text
 #: en/linkchecker.1:303
+#, fuzzy
```

Comment:
This adds a `fuzzy` marker, which means this message will _not_ be translated.

The old translation mentions Windows and Mac but omits Linux.  I suggest dropping `fuzzy` and keeping the old translation.

(Maybe we could contact the last translator -- Chris Mayo <aklhfex@gmail.com> -- and ask them to add the missing sentence about Linux systems?  Oh, wait, is that you, @cjmayo?  Do you speak German?)

Reply:
Well, I was tempted to translate that one. I guess I better had then...